content/docs/user-guide/contributing/docs.md

```diff
@@ -53,11 +53,45 @@ We will review your PR as soon as possible. Thank you for contributing!
 
 ## Development environment
 
-We highly recommend running this web app locally to check documentation or blog
-changes before submitting them, and it's quite necessary when making changes to
-the website engine itself. Source code and content files need to be properly
+We highly recommend running this web app to check documentation or blog changes
+before submitting them, and it's quite necessary when making changes to the
+website engine itself. Source code and content files need to be properly
 formatted and linted as well, which is also ensured by the full setup below.
 
+> ⚡ You can create a dev env online using Gitpod (a fully featured online IDE),
+> it'll automatically clone the repo, install the dependencies, and run the app
+> (takes a few minutes), so that you can start straight away.
+>
+> <a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
+>   <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
+> </a>
+
+<details>
+
+### How to make a Pull Request with Gitpod?
```

Comment:
@jorgeorpinel @nisarhassan12 feels it's not a good place for very long instructions like this in this doc, sorry. It's a good thing to mention that there is a possibility, may be provide the button, but docs like this should be maintained but the Gitpod, not us.

Reply:
@shcheklein I agree. Making Prs with Gitpod is briefly documented here https://www.gitpod.io/docs/pull-requests/ should I update the Pr to only have the link instead ?

Reply:
@nisarhassan12 I'll let @jorgeorpinel decide on this :)

Reply:
ok. Thanks.

Reply:
Hmm I thought the expandable section would be appropriate for this. I would just put it inside the block quote. But indeed we shouldn't have to maintain Gitpod instructions that may change in the future... Sorry, I guess I didn't think it through. Let me think... ⏳

Reply:
I guess we could just link to https://www.gitpod.io/docs/git/and https://www.gitpod.io/docs/pull-requests/ but the forking part is not mentioned anywhere unfortunately, which is my main concern.

We cold also have a separate user-guide like we have on some 3rd party tools e.g. https://dvc.org/doc/user-guide/setup-google-drive-remote or https://dvc.org/doc/user-guide/running-dvc-on-windows

I supose it'll be easier for me to take over this PR. Thanks for your contributions so far Nisar! (I hope you enabled repo maintainers to push to the PR branch 🙂)

Reply:
p.s. please notice my feature request in your repo @nisarhassan12 thanks: gitpod-io/gitpod/issues/1482

Reply:
Ok. Thanks.

Reply:
> p.s. please notice my feature request in your repo @nisarhassan12 thanks: gitpod-io/gitpod/issues/1482

@jorgeorpinel I'm not sure about this I think I have not unchecked the following while making the PR:

![image](https://user-images.githubusercontent.com/46004116/81511263-81e26580-9331-11ea-9a82-a20e4109730f.png)

Please let me know if you are not able to pus to my branch.